code/modules/client/preferences.dm

```diff
@@ -214,6 +217,9 @@ GLOBAL_LIST_EMPTY(preferences_datums)
 				else
 					dispGender = "Other"
 				dat += "<b>Gender:</b> <a href='?_src_=prefs;preference=gender'>[dispGender]</a>"
+				if(gender == PLURAL || gender == NEUTER)
+					dat += "<BR><b>Preferred Body:</b> <a href='?_src_=prefs;preference=body_type'>[body_type == MALE ? "Male" : "Female"]</a>"
+

```

Comment:
```suggestion
				dat += "<BR><b>Body type:</b> <a href='?_src_=prefs;preference=body_type'>[body_type == MALE ? "Male" : "Female"]</a>"

```
I don't like the expression "Preferred Body" here because the body you select is a fact.
Would be like calling it "Preferred Name" or "Preferred Hair Colour"

Reply:
good point, changed to Body Type